cluster/gce/util.sh

```diff
@@ -2684,15 +2699,6 @@ function kube-up() {
         create-node-port
       done
       restart_tp_scheduler_and_controller
-
-      # start-kubemark.sh sets KUBEMARK_PREFIX default to 'kubemark'
```

Comment:
is this deletion of block intended?

Reply:
This is moved to master server to reduce cluster operation from dev machine